code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -94,6 +94,7 @@
 	desc = "A special suit that protects against hazardous, low pressure environments. Has reinforced plating."
 	icon_state = "rig-mining"
 	item_state_slots = list(slot_r_hand_str = "mining_voidsuit", slot_l_hand_str = "mining_voidsuit")
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,/obj/item/weapon/pickaxe)
```

Comment:
Mining voidsuits also gets the pickaxe carrying for suit storage slot. Nice~. Will be hard for most people to make use of good seeing as they tend to connect the tank to the voidsuit. Otherwise, this PR seems good once the step_xy thing is fixed